packages/@poupe-tailwindcss/src/theme/plugin.ts

```
@@ -60,16 +64,30 @@ export function themePluginFunction(api: PluginAPI, theme: Theme): void {
+    api.addUtilities({
       [name.startsWith('.') ? name : `.${name}`]: value,
     });
+    return true;
   }
+
+  return false;
+}
+
+export function doMatchUtility(api: PluginAPI, name: string, value: CSSRuleObject): boolean {
+  const u = asMatchUtility(name, value);
+  if (u && u.name && u.value) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)